cgeglio/rancid-tomatillos
# Nav 2

### What does this PR do?
Refactor nav bar display.

### Where should the reviewer start?
Open and run web app.

### How should this be manually tested?
Upon load, verify all movies are coming in correctly and hover effects/nav looks great!

### Potential changes?
Need to implement live data from user login to nav.
